packages/amplify-category-analytics/src/commands/analytics/update.ts

```diff
@@ -43,7 +43,7 @@ export const run = async (context: $TSContext) : Promise<$TSAny> => {
     .catch(err => {
       printer.info(err.stack);
       printer.error(`There was an error updating the ${category} resource`);
-      context.usageData.emitError(err);
+      void context.usageData.emitError(err);
```

Comment:
do we ever await `emitError` ? perhaps it shouldn't return promise.

Reply:
I think it's still appropriate for it to return a promise, because it is asynchronous. This allows the caller to decide what to do with it. But, it in all cases that it is changed in this PR, I believe it is appropriate to discard the promise, since we are exiting the process in the same error handler.